src/core_codemods/lxml_safe_parser_defaults.py

```diff
@@ -0,0 +1,42 @@
+from codemodder.codemods.base_codemod import ReviewGuidance
+from codemodder.codemods.api import SemgrepCodemod
+
+
+class LxmlSafeParserDefaults(SemgrepCodemod):
+    NAME = "safe-lxml-parser-defaults"
+    REVIEW_GUIDANCE = ReviewGuidance.MERGE_AFTER_CURSORY_REVIEW
```

Comment:
This one feels like it should be `MERGE_WITHOUT_REVIEW` but I'm not sure.

Reply:
I think I agree